mw-webapp/src/component/editableText/EditableText.tsx

```diff
@@ -24,6 +24,11 @@ interface EditableTextProps<T> {
    * Additional custom class name for the editable input
    */
   className?: string;
+
+  /**
+   * Is text editable or not
```

Comment:
1. Default
2.  What does mean editable? Current mode or that doubleclick handler disabled
3.how does "or not" help?

Reply:
Added default and explained what this property means